src/scss/components/_header.scss

```diff
@@ -0,0 +1,248 @@
+                width: $width--1;
+                overflow: hidden;
+                clip: rect(1px, 1px, 1px, 1px);
+            }
+
+            .udlite-btn {
+                font-weight: $font-weight-bold;
+                line-height: $line-height--1-2;
+                letter-spacing: $letter-spacing--0-2;
+                font-size: $font-size--16;
+                position: relative;
+                align-items: center;
+                border-radius: $border-radius--4;
+                border: none;
+                cursor: pointer;
+                display: inline-flex;
+                justify-content: center;
+                user-select: none;
+                vertical-align: bottom;
+                white-space: nowrap;
+                height: $height--48;
+                color: $color-secondary;
+                background-color: transparent;
+                padding: $padding--0--12;
+                width: $width--48;
+                min-width: auto;
+            }
+        }
+    }
+
+    .teach-udemy,
+    .business-udemy {
+        a {
+            color: $color-dark-gray;
+            padding: $padding--0--12;
+            margin: $margin--0--12;
+            position: relative;
+            align-items: center;
+            border-radius: $border-radius--4;
+            border: none;
+            cursor: pointer;
+            display: inline-flex;
+            min-width: $min-width--100;
+            justify-content: center;
+            user-select: none;
+            vertical-align: bottom;
+            white-space: nowrap;
+        }
+    }
+
+    .business-udemy {
+
+        @include max-screen($width--920) {
+            display: none;
+        }
+    }
+
+    .teach-udemy {
+        @include max-screen($breakpoints-md-sm) {
+            display: none;
+        }
+    }
+
+    .udlite_btn {
+        position: relative;
+        align-items: center;
+        border-radius: $border-radius--4;
+        border: none;
+        cursor: pointer;
+        display: inline-flex;
+        min-width: $min-width--100;
+        padding: $padding--0--12;
+        justify-content: center;
+        user-select: none;
+        vertical-align: bottom;
+        white-space: nowrap;
+        font-weight: $font-weight-bold;
+        height: $height--40;
+    }
+
+    .btn-login {
+        .udlite_btn {
+            color: $color-secondary;
+            background-color: transparent;
+            border: $border-size-default solid $color-border-blue;
+        }
+    }
+
+    .btn-sign-up {
+        margin-left: $margin-left--8;
+
+        .udlite_btn {
+            background-color: $color-secondary;
+            color: $color-white;
+            height: $height--40;
+        }
+    }
+
+
+}
+
+.link-nav-bar-header {
+    position: relative;
+    border-top: $border-size-default solid $color-border-default;
+    background: #fbfbf8;
+    box-shadow: 0 2px 4px rgba(0, 0, 0, .08), 0 4px 12px rgba(0, 0, 0, .08);
```

Comment:
Move variable for this case: 
And about the color you can do like: `rgba(black, .08)`